apis/root_v3_api.go

```diff
@@ -0,0 +1,15 @@
+package apis
+
+import (
+	"net/http"
+)
+
+type RootV3Handler struct {
+	ServerURL string
+}
+
+
+func (h *RootV3Handler) RootV3GetHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.Write([]byte(`{"links":{"self":{"href":"` + h.ServerURL + `/v3"}}}`))
```

Comment:
Same as above.. I think we should have structs for these responses.

Reply:
See my other comment about this